ebsco/edsapi-koha-plugin
# Cart issues

This resolves issues with the cart not allowing you to remove items, as well as issues with the download and email functionality.

To Test:
Install the kpz.
Confirm that you can add EDS and Catalog items to cart.
Confirm that items can be removed from cart individually.
Confirm that download and send functionality works for your cart.
